packages/@uppy/drop-target/src/index.js

```diff
@@ -25,18 +27,21 @@ module.exports = class DropTarget extends BasePlugin {
     this.removeDragOverClassTimeout = null
   }
 
-  addFiles = (files) => {
-    const descriptors = files.map((file) => ({
-      source: this.id,
-      name: file.name,
-      type: file.type,
-      data: file,
-      meta: {
-        // path of the file relative to the ancestor directory the user selected.
-        // e.g. 'docs/Old Prague/airbnb.pdf'
-        relativePath: file.relativePath || null,
-      },
-    }))
+  addFiles = (files, event) => {
+    const descriptors = files.map((file) => {
+      return {
```

Comment:
Any reason not to use the shorter form as before?
```suggestion
    const descriptors = files.map((file) => ({
```

Reply:
No, I'll fix this.